### PR TITLE
Ucerf event based risk

### PR DESCRIPTION
**Pull request by micheles**
Added two UCERF calculators for event based risk, `ucerf_risk` and `ucerf_risk_fast`. For the moment only `ucerf_risk_fast` is recommended. The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/2253

### CODE REVIEW

File: openquake/calculators/ucerf_risk.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+from openquake.risklib import riskinput
+from openquake.commonlib import parallel
+from openquake.calculators import base, event_based
+from openquake.calculators.ucerf_event_based import (
+    UCERFEventBasedCalculator, DEFAULT_TRT)
+from openquake.calculators.event_based_risk import (
+    EbriskCalculator, losses_by_taxonomy)
+
+
+def compute_ruptures(sources, sitecol, gsims, monitor):
```
Comment (ptormene) [inline openquake/calculators/ucerf_risk.py]: Please add a docstring

File: openquake/calculators/event_based_risk.py [hunk 2]
```diff
@@ -731,76 +676,130 @@ class EbriskCalculator(base.RiskCalculator):
...
+    compute_ruptures = staticmethod(event_based.compute_ruptures)
+
+    # TODO: if the number of source models is larger than concurrent_tasks
+    # a different strategy should be used; the one used here is good when
+    # there are few source models, so that we cannot parallelize on those
+    def build_starmap(self, ssm, sitecol, assetcol, riskmodel, imts,
+                      trunc_level, correl_model, min_iml, monitor):
+        """
+        :param ssm: CompositeSourceModel containing a single source model
+        :param monitor: Monitor instance
```
Comment (ptormene) [inline openquake/calculators/event_based_risk.py]: Please add missing parameters to the docstring

File: openquake/calculators/ucerf_risk.py (hunk 1 above)
Comment (micheles) [inline openquake/calculators/ucerf_risk.py]: Done

File: openquake/calculators/event_based_risk.py (hunk 2 above)
Comment (micheles) [inline openquake/calculators/event_based_risk.py]: Done

**Pull request merged**